news/2656.bugfix

```diff
@@ -0,0 +1 @@
+Fixed a bug which sometimes caused pipenv to parse the ``trusted_host`` argument to pip incorrectly when parsing source URLs which specify ``verify_ssl = false``.
```

Comment:
Feel free to rewrite the fragment if you feel it is incorrect!

Reply:
I mean, while the bug may not be _caused_ by `verify_ssl` being `false` (I didn't say that it was _caused_ by setting `verify_ssl = false`, only that it sometimes occurred when parsing URLs which specified this option), it can certainly only apply to URLs that specify `verify_ssl = false` as I mention in the news fragment, wouldn't you say? 

> Fixed a bug which sometimes caused pipenv to parse the ``trusted_host`` argument to pip incorrectly

This is the nature of the bug, no?

> parsing source URLs which specify ``verify_ssl = false``.

This is a condition under which the bug occurs, right? I didn't want to get super detailed, as it's most likely an edge case that most users won't encounter -- there is a bug parsing the trusted host argument sometimes if you specify the argument via your pipfile.  There are other ways to specify trusted hosts, but this is the only one that is handled by pipenv directly, so it seems relevant to caveat this by highlighting the specific context without delving into the low level, 'if you first type x, then y, then z, then put a thing here, etc'.

Reply:
Sorry, I got totally confused about the nature of the bug, after taking a look at the code again and remembering what the bug is. I totally agree with what you wrote.